jake/command/oss.py

```diff
@@ -148,14 +148,16 @@ def handle_args(self) -> int:
                                     vector=oic_vulnerability.cvss_vector
                                 )
                             )
-
+                        cwes = None
+                        if oic_vulnerability.cwe and oic_vulnerability.cwe != "CWE-noinfo":
+                            cwes = [int(oic_vulnerability.cwe[4:])]
```

Comment:
`noinfo` might be to restrictive and hard-coded. Would it perhaps be better to just skip the cases where the CWE can't be parsed?

```suggestion
                        if oic_vulnerability.cwe and oic_vulnerability.cwe:
                            try:
                                cwes = [int(oic_vulnerability.cwe[4:])]
                            except ValueError:
                                pass    # ignore cases where conversion to int fails
```

Reply:
definitely agree, seems the cleanest solution and more bullet proof to other strange cwes.
I think you forgot to remove one check in the suggestion : 
```python
if oic_vulnerability.cwe and oic_vulnerability.cwe:
```
=> 
```python
if oic_vulnerability.cwe:
```